frontend/pages/hosts/details/HostDetailsPage/HostDetailsPage.tsx

```diff
@@ -663,7 +669,7 @@ const HostDetailsPage = ({
         hostTeamId={host.team_id}
         onSelect={onSelectHostAction}
         hostPlatform={host.platform}
-        hostStatus={host.status}
+        hostStatus={"online"}
```

Comment:
Is this hard-coded string a holdover from dev testing?

Reply:
yep. changing it back now.